--- amd-upl102/config/os-linux.h.bad	Mon Apr  7 16:41:51 1997
+++ amd-upl102/config/os-linux.h	Mon Apr  7 16:42:19 1997
@@ -252,7 +252,7 @@
 
 #define M_RDONLY 1 /* mount read-only */
 #define M_NOSUID 2 /* ignore suid and sgid bits */
-#define M_NONDEV 4 /* disallow access to device special files */
+#define M_NODEV 4 /* disallow access to device special files */
 #define M_NOEXEC 8 /* disallow program execution */
 #define M_SYNC  16 /* writes are synced at once */
 #define M_REMOUNT  32 /* alter flags of a mounted FS */
