--- template/debian/common_translation.wml	Sat Jul  6 00:07:52 2002
+++ template/debian/common_translation.wml	Sat Jul  6 00:09:28 2002
@@ -1,4 +1,12 @@
-#use wml::debian::common_tags
+<use name="intl:gettext" />
+
+<mp4h-l10n LC_MESSAGES="<or "$(CUR_LOCALE)" <downcase $(CUR_LANG) /> />" />
+<textdomain domain="templates" />
+<bindtextdomain domain="templates" path="$(ENGLISHDIR)/../locale" />
+<bind_textdomain_codeset domain="templates" codeset="<or $(CHARSET_WML) $(CHARSET) iso-8859-1 />" />
+
+#  Add <void id="foo"> to make msgids unique
+<define-tag void></define-tag>

 # Please keep slices sorted alphabetically with the exception of English.
 
@@ -24,7 +24,9 @@
   <gettext>debian, GNU, linux, unix, open source, free, DFSG</gettext>
 </define-tag>
 <define-tag returnline whitespace=delete>
-  <gettext>Back to the <a href="$(HOME)/">Debian Project homepage</a>.</gettext>
+  <printf
+      <gettext>Back to the <a href="%s">Debian Project homepage</a>.</gettext>
+      "$(HOME)/" />
 </define-tag>
 <define-tag home whitespace=delete>
   <gettext>Home</gettext>
