--- template/debian/countries.wml	Sat Jul  6 00:10:18 2002
+++ template/debian/countries.wml	Sat Jul  6 00:10:56 2002
@@ -1815,9 +1815,7 @@
     print GEN "#   File generated automatically.  Do not edit!\n";
     foreach my $c (sort keys %countries) {
         print GEN "<"."define-tag ".$c."c endtag=delete whitespace=delete>\n";
-        foreach my $t (sort keys %{$countries{$c}}) {
-            print GEN "    [".$t.":".$countries{$c}{$t}.":]\n";
-        }
+        print GEN "    <"."gettext>".$countries{$c}{EN}."<"."/gettext>\n";
         print GEN "<"."/define-tag>\n";
     }
 }
