--- template/debian/footer.wml	Sat Jul  6 01:02:43 2002
+++ template/debian/footer.wml	Sat Jul  6 01:04:22 2002
@@ -1,7 +1,9 @@
 #use wml::debian::ctime
 
 <define-tag sendcomments whitespace=delete>
-  <gettext>See the Debian <a href="$(HOME)/contact">contact page</a> for information on contacting us.</gettext>
+  <printf
+      <gettext>See the Debian <a href="%s">contact page</a> for information on contacting us.</gettext>
+      "$(HOME)/contact" />
 </define-tag>
 #<define-tag sendcomments whitespace=delete>
 #	[EN:Please send comments or corrections on these pages to:]
@@ -34,7 +36,9 @@
   <gettext>Copyright</gettext>
 </define-tag>
 <define-tag seelicense whitespace=delete>
-  <gettext>See <a href="$(HOME)/license">license terms</a></gettext>
+  <printf
+      <gettext>See <a href="%s" rel="copyright">license terms</a></gettext>
+      "$(HOME)/license" />
 </define-tag>
 
 # Do NOT translate anything below this line. If you feel something needs
