--- template/debian/languages.wml	2002-10-31 22:18:57.000000000 +0100
+++ template/debian/languages.wml	2002-10-31 22:18:19.000000000 +0100
@@ -10,6 +10,9 @@
   <gettext>How to set <a href=m4_HOME/intro/cn>the default document language</a></gettext>
 </define-tag>
 
+#   Empty slice so that WML won't fail
+[<upcase $(CUR_ISO_LANG) />::]
+
 <perl>
 # Add your language directory along with its ISO abbreviation here.
 # Do _not_ translate the directory names.
