--- template/debian/languages.wml	2002-10-31 02:01:38.000000000 +0100
+++ template/debian/languages.wml	2002-10-31 01:36:55.000000000 +0100
@@ -7,7 +7,9 @@
   <gettext>This page is also available in the following languages</gettext>
 </define-tag>
 <define-tag setlang whitespace=delete>
-   <gettext>How to set <a href=$(HOME)/intro/cn>the default document language</a></gettext>
+  <printf
+      <gettext>How to set <a href="%s">the default document language</a></gettext>
+      "$(HOME)/intro/cn" />
 </define-tag>
 
 <perl>
